### DISCUSSION TITLE
AI Seo Marketing Blog Team?

### DISCUSSION THREAD

**Discussion by Alexthekiller0311** [microsoft/autogen#230] [Q&A] [answered]
This is a prompt I wrote to bing ai however it could not create the code. Does anyone have any examples of how to create a SEO Blog Writing team for my marketing agency?

I want to create a code with agents with this reposatory https://github.com/microsoft/autogen you will tell me what code I should use. The first agent will come up with interesting topics about 10 for an online marketing agency in fluent dutch all about one topic how to improve your online presence then it will ask the second agent what 4 topics from these 10 would be the best to write a SEO optimized blog article about that agent will then pick four and then give it to the SEO Blog expert that will create those 4 blogs with optimized seo titles and descriptions then that agent will give it to the agent that proofreeds it for readibility and every agent talks to the project leader and if he agrees everything is correct it is done

**Comment by gagb** [comment, marked as answer]
This example maybe of interest to you: https://github.com/amadad/agentcy/tree/main

**Comment by Alexthekiller0311** [comment]
Wow thank you so much that is super interesting!

**Comment by brucewilson3** [comment]
Keyword volume alone won't build you a strong content team anymore. If you're building an AI SEO blog team, study how the best AI SEO services structure their content workflows around search intent instead. Honestly, HikeMyTraffic® is the best AI SEO services setup I've seen, mixing AI content tools with human editing for quality control - worth looking at a few of the best AI SEO services for inspiration before setting your own team's process.
